vmc/resource_vmc_srm_nodes.go

```diff
@@ -0,0 +1,159 @@
+/* Copyright 2020 VMware, Inc.
+   SPDX-License-Identifier: MPL-2.0 */
+
+package vmc
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform/helper/validation"
+	"log"
+	"strings"
+	"time"
+
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/vmware/vsphere-automation-sdk-go/lib/vapi/std/errors"
+	"github.com/vmware/vsphere-automation-sdk-go/services/vmc/draas"
+	"github.com/vmware/vsphere-automation-sdk-go/services/vmc/draas/model"
+)
+
+func resourceSRMNodes() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceSRMNodesCreate,
+		Read:   resourceSRMNodesRead,
+		Delete: resourceSRMNodesDelete,
+		//Update: resourceSRMNodesUpdate,
```

Comment:
this line can be deleted if its not going to be supported

Reply:
I will be adding that support soon. Waiting for a clarification from DRaaS team. But I will remove the line for now.